synapse/config/repository.py

```diff
@@ -202,6 +215,21 @@ def generate_config_section(self, data_dir_path, **kwargs):
 
         return (
             r"""
+        ##
+        ## Media Store
+        ##
+
+        # Is an external media repo in use?
+        #
+        # Set 'False' to use Synapse's media repo. If this is set to 'True',
+        # Synapse's media repo below must be disabled.
+        #
+        #external_media_repo: False
+
+        # Enable the media store service inside Synapse.
+        #
+        enable_media_repo: True
```

Comment:
config settings should be commented out, as a rule, and given an example value which might be useful (ie, the opposite to the default).